fedora-toolbox

```diff
@@ -246,6 +250,49 @@ enter()
 )
 
 
+list()
+(
+    to_print=""
+    if [ $1 = 1 ]; then
+        to_print="$to_print${LBC}Images created by fedora-toolbox${NC}\n"
+        output=$($prefix_sudo podman images \
+                --filter "label=com.redhat.component=fedora-toolbox")
+                #--format "table {{.ID}} {{.Repository}} {{.Tag}} {{.Created}}")
+        if [[ $output = "" ]]; then
+            to_print="${to_print}No images available\n"
+        else
+            to_print="$to_print$output\n"
+        fi
+    fi
+    if [ $2 = 1 ]; then
+        to_print="$to_print${LBC}Containers created by fedora-toolbox (currently running containers in green)${NC}\n"
```

Comment:
We should probably not say green, because the visible colour changes completely based on what terminal theme is being used. For example, with Solarized it turns grey.